src/components/pages/Offers/Offer/OfferDetails/OfferForm/OfferForm.jsx

```diff
@@ -33,6 +33,8 @@ import TypeTreeSelects from 'components/pages/Offers/Offer/OfferDetails/OfferFor
 import SynchronizedProviderInformation from 'components/pages/Offers/Offer/OfferDetails/SynchronizedProviderInformation'
 import { doesUserPreferReducedMotion } from 'utils/windowMatchMedia'
 
+import { CGU_URL } from '../../../../../../utils/config'
```

Comment:
On devrait normalement pouvoir écrire l'import comme ça : 

```suggestion
import { CGU_URL } from 'utils/config'
```